Core/include/Acts/Utilities/ParameterDefinitions.hpp

```diff
@@ -272,6 +311,18 @@ using SpacePointSymMatrix =
     ActsMatrix<SpacePointScalar, eSpacePointSize, eSpacePointSize>;
 using SpacePointSymMatrix = ActsSymMatrix<SpacePointScalar, eSpacePointSize>;
 
+// Matrix and vector types related to alignment parameters.
+
+using AlignmentVector =
+    ActsVector<AlignmentParametersScalar, eAlignmentParametersSize>;
+using AlignmentRowVector =
+    ActsRowVector<AlignmentParametersScalar, eAlignmentParametersSize>;
+using AlingmentMatrix =
+    ActsMatrix<AlignmentParametersScalar, eAlignmentParametersSize,
+               eAlignmentParametersSize>;
+using AlignmentToLocalCartesianMatrix =
+    ActsMatrix<AlignmentParametersScalar, 3, eAlignmentParametersSize>;
```

Comment:
By the `Local`, we just mean it's coordinates in a local frame, but does not mean it have to be 2D, right? I need 3D because the loc0, loc1 could be dependent on the local z coordinates.

Reply:
So `Local` means in the coordinate system of the surface? If so, then you can always represent a point on a surface by two dimensions. Or did I misunderstood you?

Reply:
For `PlaneSurface` and `DiscSurface`, the hit is on the local surface, i.e. the xy plane of the local frame. But for `LineSurface` , `CylinderSurface` and `ConeSurface`, this is not the case, right?

Reply:
Ah, it is a global Cartesian view of your coordinate system? Then yes, you need 3 parameters. If it would be a point that is on the surface, then it would be just 2. But now I got it